From: Andreas Henriksson <andreas@fatal.se>
Date: Mon, 2 Jan 2023 01:45:09 +0100
Subject: disable-crbug1004354

According to test/Makefile.am test-crbug1004254 should not be enabled
since it requires downloading fonts (and thus does not add it to the
TESTS variable), yet run-test.sh will explicitly run the test anyway
if it was built.... which then fails (because we don't have and can't
run curl).
---
 test/run-test.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/run-test.sh b/test/run-test.sh
index c3da72a..3af5c0b 100644
--- a/test/run-test.sh
+++ b/test/run-test.sh
@@ -429,7 +429,7 @@ rm -rf "$MYCACHEBASEDIR" "$MYCONFIG" my-fonts.conf my-out my-out.expected
 
 fi # if [ "x$EXEEXT" = "x" ]
 
-if [ -x "$BUILDTESTDIR"/test-crbug1004254 ]; then
+if false && [ -x "$BUILDTESTDIR"/test-crbug1004254 ]; then
     dotest "MT-safe global config"
     prep
     curl -s -o "$FONTDIR"/noto.zip https://noto-website-2.storage.googleapis.com/pkgs/NotoSans-hinted.zip
